### PR TITLE
Integrated logo plus minor changes 🙋🏻‍♂️

### LINKED ISSUE #7: Integrate Logo
Simple logo integration on the landing page. Should be user friendly and minimalist.

### PR DESCRIPTION
**Pull request by horikandthepurpleshrooms**
I integrated one version of logo for the app in the place of the h1 tag that kept the title from the main screen. I also added some minor changes in the `next.config.js` file. I also added two new packages, [next-compose-plugins](https://github.com/cyrilwanner/next-compose-plugins) and [next-optimized-images](https://github.com/cyrilwanner/next-optimized-images).

### CODE REVIEW

**Comment by horikandthepurpleshrooms** [comment]
image

File: pages/index.js [hunk 1]
```diff
@@ -2,15 +2,16 @@ import { Container, Tab, Row, Col, ListGroup } from "react-bootstrap";
...
 import css from "./index.css";
 
 const Home = () => (
   <Layout>
     <Container className={css.homeContainer}>
       <Row className={css.homeBoxFullFlexWrapper}>
         <Col lg={12} className={css.homeLogoWrapper}>
-          <h1>JobHub</h1>
-          <p>Minimalist Jobs Platform. We love simplicity.</p>
+        <img className={css.logo} src={logo} alt="JobHub"/>
```
Comment (rrstux) [inline pages/index.js]: Having in mind that we're using react-bootstrap, we can make use of the https://react-bootstrap.github.io/components/images/ components.

**Review by rrstux — APPROVED** [review]

File: pages/index.js (hunk 1 above)
Comment (horikandthepurpleshrooms) [reply pages/index.js]: Good, I'll make the changes in no time. 😉

Comment (horikandthepurpleshrooms) [reply pages/index.js]: Done! ✔

**Pull request merged**